jazz_scm-webhook/components/config.js

```diff
@@ -22,20 +23,19 @@
 **/
 
 var getStageConfig = (event) => {
-  var stage;
-  
-  if (event && event.awslogs && event.awslogs.data) {
-      // cw events default to dev
-      stage = 'dev';
-  }else {
-      stage = event.stage
-  } 
-  
-  var configObj;
-  
-  if (stage) {
-      configObj = require(`../config/${stage}-config.json`);
-  } 
+  var stage = event.stage;

```

Comment:
Suggestion: lines 26-38 can be reduced by using a hashmap. the performance here is better than comparing if else and the code can be concise. 

stage = { 'dev': 'dev', 'stg': 'stg', 'prod': 'prod'}
var stageConfig = stage[event.stage] ? stage[event.stage] : event;
var configObj = require(`../config/${stageConfig}-config.json`);